catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -324,12 +352,11 @@ Bundle getBundle() {
 
     public void removeContentMetadataExtractor(
             ServiceReference<ContentMetadataExtractor> contentMetadataExtractorRef) {
-        contentMetadataExtractors.remove(contentMetadataExtractorRef);
+        contentExtractors.remove(contentMetadataExtractorRef);
```

Comment:
I kept getting confused between `contentMetadataExtractors` and 'metadataExtractors`. The former scans the contents of a document to create metacard attributes. The later scans the document metadata produced by Tika to create metacard attributes. To make things less confusing, I renamed the instance variables and methods to be **content extractors** and **metadata extractors**